mkcert/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "mkcert" %}
+{% set version = "1.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}.memfault0
```

Comment:
there is no other `mkcert` conda package but i kindof like namespacing ours with the version tag here